### PR TITLE
docs: add new image directive snippet 

### PR DESCRIPTION
**Pull request by deepakrudrapaul**
Hey this PR allows to show a example of how to use the _NgOptimizedImage_ directive

### CODE REVIEW

File: src/pages/snippets/v15-new-image-directive.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### Import NgOptimizedImage
+
+```typescript
+import { NgOptimizedImage } from "@angular/common";
+```
+
+### Enable the directive
+
+To enable the NgOptimizedImage directive, replace your image's src attribute with _ngSrc_ and to prevent image related layout shifts,
+NgOptimizedImage requires that you specify a height and width for the image.
```
Comment (yharaskrik) [inline src/pages/snippets/v15-new-image-directive.mdx:21]: Might want to add that height and width is optional if using fill mode

https://angular.io/guide/image-directive#using-fill-mode

File: src/pages/snippets/v15-new-image-directive.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: New Image Directive in Angular v15
+description: "Example of New Image Directive in Angular 15 "
+tags: ["angular15", "image directive"] # comma separated list which gets trimmed
+pubDate: Mar 5, 2023
+contributedBy: "@deepakrudrapaul"
+---
+
+Here is a quick example of how to use the _NgOptimizedImage_ directive.
```
Comment (yharaskrik) [inline src/pages/snippets/v15-new-image-directive.mdx:9]: I would suggest saying this is a `simple` example as there is a LOT of configuration possible that will be shown as console warnings when the app starts up

https://angular.io/guide/image-directive#performance-features
https://angular.io/guide/image-directive#configuring-an-image-loader-for-ngoptimizedimage

File: src/pages/snippets/v15-new-image-directive.mdx (hunk 1 above)
Comment (deepakrudrapaul) [reply src/pages/snippets/v15-new-image-directive.mdx]: Thank you for the suggestion

File: src/pages/snippets/v15-new-image-directive.mdx (hunk 2 above)
Comment (deepakrudrapaul) [reply src/pages/snippets/v15-new-image-directive.mdx:9]: Good idea changed it _simple_. Thanks

Comment (santoshyadavdev) [reply src/pages/snippets/v15-new-image-directive.mdx:9]: I can still see this as quick.

**Review by santoshyadavdev — CHANGES_REQUESTED** [review]

File: src/pages/snippets/v15-new-image-directive.mdx (hunk 2 above)
Comment (deepakrudrapaul) [reply src/pages/snippets/v15-new-image-directive.mdx]: I have pushed the changes. Thanks

**Review by santoshyadavdev — APPROVED** [review]
Thank you 💙

**Comment by santoshyadavdev** [comment]
@all-contributors add @deepakrudrapaul for docs

**Pull request merged**